gef-ui/server/utils/helpers.js

```diff
@@ -173,14 +183,24 @@ const previewItemConditions = (previewParams) => {
      * creates change link with different href to change to unissued again for the stage row
      */
     if (item.id === 'hasBeenIssued') {
-      summaryItems = previewSummaryItems(issuedToUnissuedHref, changedToIssueShow, item);
+      summaryItems = generateActionsArrayForItem({
+        href: issuedToUnissuedHref,
+        visuallyHiddenText: item.label,
+        actionType: shouldDisplayChangeLinkIfIssued && 'Change',
```

Comment:
Is it worth pulling out the text 'Change' and 'Add' as a constant somewhere (maybe just at the top of the file if it's only used in this file)